pkg/features/list.go

```diff
@@ -44,4 +44,7 @@ var (
 
 	// CentralEvents enables APIs (including collection) and UI pages for Central events.
 	CentralEvents = registerFeature("Enable Central events", "ROX_CENTRAL_EVENTS", false)
+
+	// ScannerV4 use Scanner V4 as the default Stackrox Scanner across Central and Secured Clusters.
+	ScannerV4 = registerFeature("Enables ScannerV4 as the default stackrox image scanner", "ROX_SCANNERV4_ENABLE", false)
```

Comment:
The intention behind this PR is to release Scanner V4 enabled. I don't believe we are planning to support the lifecycle of enabling/disabling with env vars. It might be that the definition of a tech preview release is supporting enabling/disabling the feature. I will get more information or ask people to answer here.

Reply:
In any case, moving this to an env var would be easy and I don't think this blocks the PR at the moment.

Reply:
I think we should make sure people can enable/disable scanner v4 since some features such as detecting nodejs vulns is not available in scanner v4.